### ISSUE TITLE
Slipstream the extended kernel into a Windows 7 x64 WIM

### ISSUE THREAD

**Issue by RetroSoftwareRepository** [vxiiduu/VxKex#48]
Can it be done? If so can support for doing this be added? Thanks!

**Comment by vxiiduu** [comment]
Yes it is theoretically possible. However, I will not bother to explicitly support this scenario. All you would have to do is have one of your post install scripts run the vxkex installer. In the new version of the installer I will add some commandline switches to make silent and automated installs easier to do.

**Comment by Saswat2005** [comment]
And what I am going to make an extended kernel of Windows 7 just like Windows Vista Extended Kernel what made.
